rapids-cmake/cpm/nvcomp.cmake

```diff
@@ -120,6 +109,14 @@ function(rapids_cpm_nvcomp)
     endif()
   endif()
 
+  include("${rapids-cmake-dir}/cpm/detail/generate_patch_command.cmake")
+  rapids_cpm_generate_patch_command(nvcomp ${version} patch_command)
+
+  # Apply any patch commands to the proprietary binary
+  if(nvcomp_proprietary_binary AND patch_command)
+    execute_process(COMMAND ${patch_command} WORKING_DIRECTORY ${nvcomp_ROOT})
```

Comment:
I think I'm missing something. Where is the nvcomp patch coming from?

Reply:
Line 113 has the call to `rapids_cpm_generate_patch_command` which will populate the `patch_command` variable

Reply:
Right, but I don't actually see any patches for nvcomp in the versions.json file. Are we planning to put that in per-package override.json files?

Reply:
Ohhh.

So originally this PR moved the inline logic for patching `2.3.2` to a patch file. While this PR was open we moved 22.10 to nvcomp 2.4 making the patch for 2.3.2 not necessary, and therefore it was dropped from this PR.

So that is why in this PR we see the inline logic for 2.3.2 being removed and no matching patch being added.

Reply:
Got it, so should we remove the nvcomp patching logic for now then, or do you want to keep the framework in place in anticipation of needing future patches?

Reply:
I would prefer to keep the framework in place. That way we can use patches in override json files without any modifications to rapids-cmake itself.